docs/src/reference/python/operations/slice.rst

```diff
@@ -0,0 +1,6 @@
+slice
```

Comment:
Could you just capitalize this here. Just to be consistent,

```suggestion
Slice
```

Reply:
Consistent with what? Granted there is capitalisation in index.rst and samples_reduction.rst, but not in allclose.rst, dot.rst, lstsq.rst, remove_gradients.rst, or solve.rst

Reply:
Ahh okay we can also leave it. I just saw the header of `Operations`